common/build/eslint-config-fluid/minimal-deprecated.js

```diff
@@ -249,13 +249,7 @@ module.exports = {
 
 		// #region FORMATTING RULES
 
-		"@typescript-eslint/brace-style": [
-			"error",
-			"1tbs",
-			{
-				allowSingleLine: true,
-			},
-		],
+		"@typescript-eslint/brace-style": "off",
```

Comment:
We should leave a comment noting why this is disabled.

Reply:
I'm also surprised by this. Does Biome create single-line scope blocks? Can we configured it not to?

Reply:
Here's the motivating example I found:

```ts
export class DiceRollerController
	extends events_pkg.EventEmitter
	implements IDiceRollerController
{
```

I don't know if it's a biome bug or not. Regardless, though, I think we should try to remove formatting rules from the lint config wherever possible to improve lint performance and not worry about conflicting behaviors. We've already done that with ost of them, but I think we also relied on the prettier eslint config to disable some rules for us. That's still in place, so presumably there won't be many beyond this one that conflict.